### DISCUSSION TITLE
Visual updates

### DISCUSSION THREAD

**Discussion by aly-fly** [Edzelf/ESP32Radio-V2#144] [Ideas]
Hi, thanks for the great project.
I made a few tweaks to make it more user friendly. Let me know if you are interested in any of the following. If yes, I can prepare a pull request(s).
- Bigger buttons on the web page for easier manipulation on the mobile phone.
- Additional search buttons for seasonal themes, like Halloween, Xmas, ...
- Bigger font for important messages. One more line. Split artist and title. On "bluetft" display.
- Verbose startup messages and colored error messages on startup.

**Comment by aly-fly** [comment]
![20240325_204906](https://github.com/Edzelf/ESP32Radio-V2/assets/98057593/804cd675-181b-426a-b17e-1512bb56f693)

**Comment by akaminski22** [comment]
> Hi, thanks for the great project. I made a few tweaks to make it more user friendly. Let me know if you are interested in any of the following. If yes, I can prepare a pull request(s).
> 
> * Bigger buttons on the web page for easier manipulation on the mobile phone.
> * Additional search buttons for seasonal themes, like Halloween, Xmas, ...
> * Bigger font for important messages. One more line. Split artist and title. On "bluetft" display.
> * Verbose startup messages and colored error messages on startup.

Hi,
Can you share yours code?
